web_full_stack/ruby/README.md

```diff
@@ -3,7 +3,10 @@
 This is the Ruby/Rails part of the code extension. Tests are written using Rspec.
 
 ## Setup
-1. Install ruby 2.7.2. We like to use [rbenv](https://github.com/rbenv/rbenv), but do your thing, it's all good.
+
+**During the interview, this application will run in a cloud-based IDE, so don't worry about running it locally. These setup instructions are purely for documentation purposes only.**
+
+1. Install ruby 2.7.2. We like to use [rbenv](https://github.com/rbenv/rbenv).
```

Comment:
looks like this got bumped to 3.1.3
https://github.com/reverbdotcom/code_extension/commit/af5d12e71ea07d3e65b22af3ba626c6e6cde80c8#diff-8f035957771fb454a75b7775adc4ea7af4770b2080279b73c6b7ec6bb678b84d. maybe it can be generalized?
```suggestion
1. Install Ruby based on the `.ruby-version` file. We like to use [rbenv](https://github.com/rbenv/rbenv).
```

Reply:
ope yep. looks like i was on the outdated master branch. but yeah, i can generalize this.